src/pam-config.8.xml

```diff
@@ -201,6 +201,22 @@
               </para>
             </listitem>
           </varlistentry>
+          <varlistentry>
+            <term><option>--kanidm</option></term>
+            <listitem>
+              <para>
+                Enable/Disable pam_kanidm.so
+              </para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term><option>--kanidm-debug</option></term>
```

Comment:
These are currently in alphabetical order, but you've placed kanidm at the beginning of the list. IIRC, this does effect the output.

Reply:
🤔 himmelblau was also placed at the very beginning too?

Reply:
pam-config.8.xml is autogenerated.
The list in pam-modules.h should be in alphabetical order, for some lists that's not possible if there are dependencies to other modules.